packages/components-dev/ssr/breadcrumbs-ssr-test/breadcrumbs-ssr-test.ts

```diff
@@ -0,0 +1,65 @@
+import { AsyncPipe } from '@angular/common';
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { RouterLink } from '@angular/router';
+import { KbqBreadcrumbsModule } from '@koobiq/components/breadcrumbs';
+import { KbqButtonModule } from '@koobiq/components/button';
+import { KbqTopBarModule } from '@koobiq/components/top-bar';
+import { of, timer } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+/**
+ * @title Breadcrumbs ssr test
+ */
+@Component({
+    standalone: true,
+    selector: 'dev-breadcrumbs-ssr-test',
+    template: `
+        <kbq-top-bar>
+            @if (breadcrumbItems$ | async; as breadcrumbs) {
+                <nav kbq-breadcrumbs wrapMode="none">
+                    @for (breadcrumb of breadcrumbs; track breadcrumb.label; let last = $last) {
+                        <kbq-breadcrumb-item [routerLink]="breadcrumb.url" [text]="breadcrumb.label">
+                            <a *kbqBreadcrumbView [routerLink]="breadcrumb.url" tabindex="-1">
+                                <button
+                                    [disabled]="last"
+                                    [attr.aria-current]="last ? 'page' : null"
+                                    kbq-button
+                                    kbqBreadcrumb
+                                >
+                                    <span>{{ breadcrumb.label }}</span>
+                                </button>
+                            </a>
+                        </kbq-breadcrumb-item>
+                    }
+                </nav>
+            }
+        </kbq-top-bar>
+    `,
+    imports: [
+        RouterLink,
+        KbqBreadcrumbsModule,
+        KbqButtonModule,
+        AsyncPipe,
+        KbqTopBarModule
+    ],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class DevBreadcrumbsSsrTest {
```

Comment:
чтобы явно было понятно, что проверяется в данном компоненте
```suggestion
export class DevBreadcrumbsHydration {
```